### PR TITLE
[APP-534] Fixes related to Clearbit

### PR DESCRIPTION
**Pull request by hugomano**
### Fixes
- Revert `incrementOriginResultID` for `originResultID`: async messages channels was handled with the wrong callback
- Fix message passing from the BG to the CS
- Fix `chrome.tabs.create` optional cb

### Todo
- Publish as `4.0.2` and make PR in Bx

### CODE REVIEW

File: src/browser/chrome-extension.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ ipcMain.on(constants.RUNTIME_SENDMESSAGE, function (event, extensionId, message,
 
   page.webContents.sendToAll(`${constants.RUNTIME_ONMESSAGE_}${extensionId}`, event.sender.id, message, resultID)
   ipcMain.once(`${constants.RUNTIME_ONMESSAGE_RESULT_}${resultID}`, (eventResult, result) => {
-    eventResult.sender.send(`${constants.RUNTIME_SENDMESSAGE_RESULT_}${originResultID}`, result)
+    event.sender.send(`${constants.RUNTIME_SENDMESSAGE_RESULT_}${originResultID}`, result)
```
Comment (magne4000) [inline src/browser/chrome-extension.js]: Perhaps you should rename `event` and `eventResult` to better match their origin

File: src/renderer/chrome-api.js [hunk 2]
```diff
@@ -140,7 +141,7 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
   subscribeAndForwardEvents(chrome);
 
   if (!isPackaged) {
-    chrome = loggingProxy(chrome);
+    // chrome = loggingProxy(chrome);
```
Comment (magne4000) [inline src/renderer/chrome-api.js]: ?

File: src/renderer/logging-proxy.ts [hunk 3]
```diff
@@ -56,7 +56,7 @@ const handler = {
 
     if (['string', 'number'].includes(typeof apis[prop])) {
       const result = apis[prop];
-      log(`${apis.__path}.${prop} `, result);
+      // log(`${apis.__path}.${prop} `, result);
```
Comment (magne4000) [inline src/renderer/logging-proxy.ts]: Delete it?

Comment (hugomano) [reply src/renderer/logging-proxy.ts]: uncommented lines

File: src/renderer/chrome-api.js (hunk 2 above)
Comment (hugomano) [reply src/renderer/chrome-api.js]: even in dev env, we don't want logs by default, it's the entry point for log all API calls

File: src/browser/chrome-extension.js (hunk 1 above)
Comment (hugomano) [reply src/browser/chrome-extension.js]: done

**Pull request merged**